### PR TITLE
[docs] Updates to TTS task guide with regards to the new TTS pipeline 

### PR DESCRIPTION
**Pull request by MKhalusova**
This PR adds: 

- An example of inference with the newly added text-to-speech pipeline
- A necessary training code update to make sure the fine-tuned checkpoint is usable via the pipeline (saving the processor)
- An introduction example of inference with Bark and a link to the course for more examples

### CODE REVIEW

File: docs/source/en/tasks/text-to-speech.md [hunk 1]
```diff
@@ -493,29 +523,70 @@ Push the final model to the 🤗 Hub:
...
+```py
+>>> forward_params = {"speaker_embeddings": speaker_embeddings}
+>>> output = pipe(text, forward_params=forward_params)
+>>> output
+{'audio': array([-6.82714235e-05, -4.26525949e-04,  1.06134125e-04, ...,
+        -1.22392643e-03, -7.76011671e-04,  3.29112721e-04], dtype=float32),
+ 'sampling_rate': 16000}
+```
+
+Here's a code snippet you can use to listen to the resulting audio in a notebook:
```
Comment (ylacombe) [inline docs/source/en/tasks/text-to-speech.md:562]: (nit) invert the text before the Audio code snippet

```suggestion
You can then listen to the result:
```

**Review by ylacombe — APPROVED** [review]
LGTM ! Thanks for updating it

I've left some really small suggestions, feel free to ignore them!

**Review by amyeroberts — APPROVED** [review]
Looks great - thanks for adding this!

File: docs/source/en/tasks/text-to-speech.md [hunk 2]
```diff
@@ -19,16 +19,40 @@ rendered properly in your Markdown viewer.
 [[open-in-colab]]
 
 Text-to-speech (TTS) is the task of creating natural-sounding speech from text, where the speech can be generated in multiple 
-languages and for multiple speakers. The only text-to-speech model currently available in 🤗 Transformers 
-is [SpeechT5](model_doc/speecht5), though more will be added in the future. SpeechT5 is pre-trained on a combination of 
+languages and for multiple speakers. Several text-to-speech models are currently available in 🤗 Transformers, such as 
+[Bark](../model_doc/bark), [MMS](../model_doc/mms), and [SpeechT5](../model_doc/speecht5).
```
Comment (Vaibhavs10) [inline docs/source/en/tasks/text-to-speech.md:23]:
```suggestion
[Bark](../model_doc/bark), [MMS](../model_doc/mms), [VITS](../model_doc/vits) and [SpeechT5](../model_doc/speecht5). 
```

**Review by Vaibhavs10 — APPROVED** [review]
Thank you for adding this @MKhalusova - I was going to add the same and saw that you've already done it. Just a wee suggestion to add info about `VITS` as well.

Thank you! ♥️

File: docs/source/en/tasks/text-to-speech.md [hunk 3]
```diff
@@ -19,16 +19,40 @@ rendered properly in your Markdown viewer.
 [[open-in-colab]]
 
 Text-to-speech (TTS) is the task of creating natural-sounding speech from text, where the speech can be generated in multiple 
-languages and for multiple speakers. The only text-to-speech model currently available in 🤗 Transformers 
-is [SpeechT5](model_doc/speecht5), though more will be added in the future. SpeechT5 is pre-trained on a combination of 
+languages and for multiple speakers. Several text-to-speech models are currently available in 🤗 Transformers, such as 
+[Bark](../model_doc/bark), [MMS](../model_doc/mms), [VITS](../model_doc/vits) and [SpeechT5](../model_doc/speecht5). 
+
+You can easily generate audio using the `"text-to-audio"` pipeline (or its alias - `"text-to-speech"`). Some models, like Bark,
```
Comment (sanchit-gandhi) [inline docs/source/en/tasks/text-to-speech.md:25]: Nice!

**Review by sanchit-gandhi — APPROVED** [review]
Very cool - thanks for this update @MKhalusova!

File: docs/source/en/tasks/text-to-speech.md [hunk 4]
```diff
@@ -485,6 +509,12 @@ the `per_device_train_batch_size` incrementally by factors of 2 and increase `gr
 >>> trainer.train()
 ```
 
+To be able to use your checkpoint with a pipeline, make sure to save the processor with the checkpoint: 
+
+```py
+>>> processor.save_pretrained("YOUR_ACCOUNT_NAME/speecht5_finetuned_voxpopuli_nl")
```
[the remark this answers is not in the document]
Comment (MKhalusova) [reply docs/source/en/tasks/text-to-speech.md:515]: I haven't tested, maybe. I think this way, it's rather consistent and definitely works :)

**Pull request merged**